rpmlb/builder/koji.py

```diff
@@ -20,8 +20,9 @@ def __init__(
         *,
         koji_scratch: bool = False,
         koji_epel: int = None,
-        koji_target_format: str = DEFAULT_TARGET_FORMAT,
-        koji_profile: Optional[str] = None
+        koji_target_format: str = None,
+        koji_profile: Optional[str] = None,
+        **extra_options
```

Comment:
Nowhere. It needs to be present in the signature, so we would not get "unexpected keyword argument" errors.

I know you were trying to get rig of unused parameters with some tool – if you know of a way to mark the argument as "unused, but it needs to be there", let me know.